### PR TITLE
update Ephor playbook and Rstudio roles 

### PR DESCRIPTION
**Pull request by jelletreep**
@dometto, i made some updates for the Ephor project. Because I added the "iRods tools " and "r workbench" components, I could remove some redundant roles from this playbook. Note that the Rstudio role is also updated. I removed the installation of R because it is done in r workbench, and centOS installation because centOS is not supported anymore if I am correct. 
Any idea if the R studio role is used elsewhere? Is it an idea to add Rstudio to r workbench by default (for desktop workspaces)? @chStaiger do you think it is useful for Emilia's project as well?

I created a workspace and everything seems to work as expected!

### CODE REVIEW

File: playbooks/ephor.yml [hunk 1]
```diff
@@ -2,11 +2,7 @@
 - name: Workspace playbook for Ephor use-case
   hosts: localhost
   roles:
-    - irods_repo
-    - irods_icommands
     - irods_iselect
```
Comment (dometto) [inline playbooks/ephor.yml:5]: Is iselect still needed?

File: playbooks/ephor.yml [hunk 2]
```diff
@@ -2,11 +2,7 @@
...
-    - irods_skel
     - myrods-sync
```
Comment (dometto) [inline playbooks/ephor.yml:6]: Is myrods-sync still needed?

**Review by dometto — APPROVED** [review]
Looks good to me! Is this a good moment to add some documentation for the `rstudio` role, in which it's made explicit that R is already expected to be present? (Maybe it's there but I don't see it on the index page of our documentation.)

File: playbooks/ephor.yml (hunk 2 above)
Comment (jelletreep) [reply playbooks/ephor.yml:6]: Yes, iselect and myrods-sync are currently used by the data manager to transfer data to yoda. If they started working with ibridges we can remove these.

**Comment by jelletreep** [comment]
> Looks good to me! Is this a good moment to add some documentation for the `rstudio` role, in which it's made explicit that R is already expected to be present? (Maybe it's there but I don't see it on the index page of our documentation.)

Yes, good point, I will!

**Comment by dometto** [comment]
I think it's a great idea to add RStudio to the `r-workbench` playbook, on desktop workspaces. I think this could be very simply done by adding something like the following to the `roles:` section of the `r-workbench` playbook:

```yaml
roles:
  - fact_workspace_info
  - rstudio
    when: fact_desktop_workspace
```

...although there may be the complication that roles are executed before tasks, so we should make sure `rstudio` is included after r is installed. In that case, you might have to do:

```
roles:
  - fact_workspace_info
 tasks:
     # R is installed here
    - name: Include ansible-r role
      include_role:
        role: rstudio
```

**Comment by jelletreep** [comment]
I will merge this one, and create a new issue for moving/adding Rstudio installation to r workbench

**Pull request merged**